src/frontend/Komponenter/Behandling/Inngangsvilkår/NyttBarnSammePartner/TidligereVedtaksperioderSøkerOgAndreForeldre.tsx

```diff
@@ -100,10 +100,10 @@ const TidligereVedtaksperioderSøkerOgAndreForeldre: FC<{
                 }
                 return (
                     <TabellVisning
-                        ikon={TabellIkon.REGISTER}
```

Comment:
Burde vi ikke ha registerikone for denne? 👀  Kan det egentlige være en feil i skissene?

Reply:
Dette er saksopplysninger og ikke registerdata, så den skal være uten ikon frem til vi evt. får et ikon for saksopplysninger☺️